pandas/core/base.py

```diff
@@ -7,8 +7,10 @@
 from pandas.core import common as com
 import pandas.core.nanops as nanops
 import pandas.lib as lib
+from pandas.compat.func_defaults import TRANSPOSE_DEFAULTS
```

Comment:
so in instead of these imports, so the following.

```
from pandas.compat.numpy import function as numpy_compat

def transpose(self, *args,  **kwargs):
      .....
      numpy_compat.transpose(*args, **kwargs)
```

It might be a bit odd to actually call ``validate_transpose`` but I think its simpler.